File: Demo-LLP-MIP/Demo-LLP-MIP-PHP/v1/isign/argument/verify/VerifyParams.php
<?php


class VerifyParams
{
    public $mch_id = null;
    public $sub_mchid = null;
    public $user_id = null;
    public $token = null;
    public $verify_code = null;

    public function to_dict() {
        $data = [];
        foreach ($this as $key => $value) {
            if ($value !== null) {
                $data[$key] = $value;
            }
        }
        return json_encode($data);
    }
}

File: Demo-LLP-MIP/Demo-LLP-MIP-Python/v1/isign/argument/verify/VerifyParams.py
class VerifyParams:
    def __init__(self):
        self.mch_id = None
        self.sub_mchid = None
        self.user_id = None
        self.token = None
        self.verify_code = None

    def to_dict(self):
        return {
            "mch_id": self.mch_id,
            "sub_mchid": self.sub_mchid,
            "user_id": self.user_id,
            "token": self.token,
            "verify_code": self.verify_code
        }